db/migrate/20150730052415_remove_translations_from_spree_tables.spree_i18n.rb

```diff
@@ -0,0 +1,66 @@
+# This migration comes from spree_i18n (originally 20150609154031)
+class RemoveTranslationsFromSpreeTables < ActiveRecord::Migration
+  def up
+    # Don't migrate if we still use Globalize, i.e. through spree_globalize Gem
+    return if defined?(Globalize)
+
+    %w(
+      OptionType
+      OptionValue
+      ProductProperty
+      Product
+      Promotion
+      Property
+      Store
+      Taxon
+      Taxonomy
+    ).each do |class_name|
+      migrate_translation_data!(class_name)
+    end
+  end
+
+  def down
+    return if defined?(Globalize)
+    raise ActiveRecord::IrreversibleMigration
+  end
+
+  private
+
+  def current_locale
+    I18n.default_locale || 'en'
+  end
+
+  def migrate_translation_data!(class_name)
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for migrate_translation_data! is too high. [21.4/15]